lib/config/ChainConfigManager.ts

```diff
@@ -0,0 +1,293 @@
+import { ChainId } from '@uniswap/sdk-core';
+import { RoutingType } from '../constants';
+
+type ChainConfig = {
+  routingTypes: (
+    | {
+        routingType: RoutingType.CLASSIC | RoutingType.DUTCH_V2 | RoutingType.RELAY;
+        skipRFQ?: boolean;
```

Comment:
Relay order have no notion of RFQ so could separate into a separate type here

Reply:
Good call. Same with Classic.